package.json

```diff
@@ -279,11 +281,15 @@
     "outputName": "test-results.xml"
   },
   "scripts": {
+    "postinstall": "npm run build --workspaces",
     "check": "eslint --ext .js,.jsx,.tsx,.ts . --quiet --cache && stylelint \"**/*.{css,scss}\" --cache",
     "fix": "eslint --ext .js,.jsx,.tsx,.ts . --quiet --fix --cache && stylelint \"**/*.{css,scss}\" --fix --cache",
-    "build": "cross-env NODE_ENV=production webpack",
-    "run": "webpack --progress --watch",
-    "dev-server": "webpack serve --mode development",
+    "build": "node scripts/build",
```

Comment:
This is a pretty big change since we're now using dedicated Node.js scripts to build the web app instead of trying to fit this all in here. I did that so that we can use concurrently more easily without fighting with its kind-of-messy command line interface